elements/lisk-api-client/src/types.ts

```diff
@@ -100,11 +100,26 @@ export interface NodeInfo {
 	readonly networkIdentifier: string;
 	readonly lastBlockID: string;
 	readonly height: number;
+	readonly genesisHeight: number;
 	readonly finalizedHeight: number;
 	readonly syncing: boolean;
 	readonly unconfirmedTransactions: number;
 	readonly genesisConfig: GenesisConfig;
 	readonly registeredModules: RegisteredModule[];
+	readonly network: {
```

Comment:
maybe include this in this issue as well?

Reply:
Updated the the issue.